### ISSUE TITLE
o-forms: duplicate ID used in demo

### ISSUE THREAD

**Issue by JakeChampion** [Financial-Times/origami#528]
# Issue
This is from page 82 of the [audit report](https://drive.google.com/file/d/17wq6b0CIX41RA4muBFDzUDxE0-qzHBIH/view)

Issue can be found on: <https://www.ft.com/__origami/service/build/v3/demo?component=o-forms@9.0.1&demo=inverse-form&system_code=origami-repo-data&brand=master>

The date input fields have not been grouped together in a fieldset and legend and have been labelled using aria-labelledby. However, a duplicate ID value has been used on both the date inputs and the radio buttons below them.
As a result, screen reading software uses the ‘Date input’ label on both the date inputs and the radio buttons when navigating out of the context of the page, and the radio buttons appear as unlabelled when navigating in the context of the page.
This issue may also affect voice activation users who will be unable to reference the radio buttons by their visual label.

HTML which has the duplicate ID and aria-labelledby attributes:
> &lt;div class="o-forms-field o-forms-field--optional o-forms-field--inverse" role="group" **aria-labelledby="date-group-title"**&gt;
> 	&lt;span class="o-forms-title"&gt;
> 		&lt;span class="o-forms-title__main" **id="date-group-title"**&gt;Date input&lt;/span&gt;
> 		&lt;span class="o-forms-title__prompt"&gt;Optional prompt text&lt;/span&gt;
> 	&lt;/span&gt;
> ...
> &lt;/div&gt;
> 
> &lt;div class="o-forms-field o-forms-field--inverse" role="group" **aria-labelledby="date-group-title"**&gt;
> 	&lt;span class="o-forms-title" aria-hidden="true"&gt;
> 		&lt;span class="o-forms-title__main" **id="date-group-title"**&gt;Radio box input&lt;/span&gt;
> 	&lt;/span&gt;
> 
> ...
> &lt;/div&gt;

 
# Proposed Solution:
Ensure the radio inputs have a unique ID attribute which matches the aria-labelledby attribute for the group. This will enable users of assistive technologies to identify and interact with the radios correctly.

**Comment by JakeChampion** [comment]
This was fixed in https://github.com/Financial-Times/origami/commit/034f1f755ad6ea7f6ff2adc7df397c0bde050e40
